enterprise/backend/src/metabase_enterprise/sandbox/api/gtap.clj

```diff
@@ -20,11 +20,10 @@
     (db/select-one GroupTableAccessPolicy :group_id group_id :table_id table_id)
     (db/select GroupTableAccessPolicy {:order-by [[:id :asc]]})))
 
-#_{:clj-kondo/ignore [:deprecated-var]}
-(api/defendpoint-schema GET "/:id"
+(api/defendpoint GET "/:id"
   "Fetch GTAP by `id`"
   [id]
-  (api/check-404 (GroupTableAccessPolicy id)))
+  (api/check-404 (db/select-one GroupTableAccessPolicy :id id)))
```

Comment:
Invoking a keyword with an integer will just give you `nil` in Toucan 2, so this won't work anymore going forward. Replaced with an equivalent form.